src/Utilities/Compiler/WellKnownTypeNames.cs

```diff
@@ -252,7 +252,7 @@ internal static class WellKnownTypeNames
         public const string SystemThreadingCancellationToken = "System.Threading.CancellationToken";
         public const string SystemThreadingInterlocked = "System.Threading.Interlocked";
         public const string SystemThreadingMonitor = "System.Threading.Monitor";
-        public const string SystemThreadingTasksGenericTask = "System.Threading.Tasks.Task`1";
+        public const string SystemThreadingTasksTasks = "System.Threading.Tasks.Task`1";
```

Comment:
This should be `SystemThreadingTasksTask1`
```suggestion
        public const string SystemThreadingTasksTask1 = "System.Threading.Tasks.Task`1";
```